### DISCUSSION TITLE
Unable to capture signatures

### DISCUSSION THREAD

**Discussion by bholmesaco** [sheltermanager/asm3#1575] [Q&A / Help]
No option to sign documents, nor capture staff signatures. I tried looking in the FAQs and the manual but wasn't able to find anyone else with the same problem. No idea what I did wrong.

![Screenshot 2024-11-18 at 2 13 37 PM](https://github.com/user-attachments/assets/cb8dcb7a-88d6-4df1-b73c-5932204a7d73)
<img width="743" alt="Screenshot 2024-11-18 at 2 13 16 PM" src="https://github.com/user-attachments/assets/cc3bd170-332f-4f53-9c96-e85ffe08e177">

**Comment by bobintetley** [comment]
You didn't do anything wrong, signatures are disabled by default. You can enable them by adding this line to your asm3.conf file:

electronic_signatures = touch

**Reply by bholmesaco** [reply]
Hey bobintetley, thank you for the response! I found that line and uncommented the line to enable it, but I am still not able to capture/add signatures. I appreciate any further insight you might be able to provide.
<img width="601" alt="Screenshot 2024-11-28 at 5 28 44 AM" src="https://github.com/user-attachments/assets/6ab9cd12-34b1-414f-a808-e47debfd5814">

**Reply by bholmesaco** [reply]
still unable to get this to work

**Comment by bholmesaco** [comment]
still unable to capture or use any signatures :(

**Comment by bobintetley** [comment]
Do you have extra spaces on the end of the line in the config file after the word touch maybe? The only reason for those fields to disappear is if that value is wrong. Maybe your asm install isn't using the config file you think it is? It's typically the one in /etc/asm3.conf

**Comment by bholmesaco** [comment]
Thank you! I wasn't able to get it myself, but the hosting company was able to get it done with that information. Thank you so much! Definitely a game changer!!

**Reply by bobintetley** [reply]
Thanks for the update, just out of interest who is the hosting company?

**Comment by bholmesaco** [comment]
The hosting company is https://www.m3server.com/ and I have the script running here [redacted]

**Reply by bobintetley** [reply]
Thanks for the info.
